### ISSUE TITLE
remove Reminder Cog

### ISSUE THREAD

**Issue by rtk-rnjn** [quotientbot/Quotient-Bot#130]
The Reminder Cog is a cog that does not have any commands or events.

I propose that we completely remove the Reminder Cog and transfer all of its functions and methods to the bot core. This will simplify the codebase and make it easier to maintain.

---

Benefits
- Simplification of the codebase
- Easier maintenance
- Reduced risk of bugs

Risks
- Potential for downtime

---

I believe that the benefits of removing the Reminder Cog outweigh the risks.

**Comment by deadaf** [comment]
yes this actually makes sense, a lot of functionalities depends on the reminder cog at the moment and if for any reason this cog isn't loaded, a lot of things will break. 
I will look into it.
